### ISSUE TITLE
Wallhack + Ghost vision , или же читы для SS13

### ISSUE THREAD

**Issue by AzzyDreemurr13** [TauCetiStation/TauCetiClassic#12069]
#### Подробное описание проблемы
Особое внимание для Билдмейнтеров и хоста
@volas , @KIBORG04 

https://www.youtube.com/watch?v=YMAALZr1JYA

#### Что должно было произойти
![IMG_20230912_085937](https://github.com/TauCetiStation/TauCetiClassic/assets/74739991/f67ffd16-9210-4962-93f6-1eb1daebb381)

#### Что произошло на самом деле

Человек видет через стены и гостоту 

#### Как повторить
Всё описано в видео + закреп комментарий

#### Дополнительная информация:
![IMG_20230912_084413](https://github.com/TauCetiStation/TauCetiClassic/assets/74739991/c8b2faf3-62d1-4570-ac37-fac7e55294c6)
Слили информацию об этом в Аппеляции на форуме:
https://forum.taucetistation.org/t/apellyacziya-tankistpidoras228-l4rever/36880?u=azzy.dreemurr

**Comment by KIBORG04** [comment]
Читы для сс13 и так любой школьник делает

**Comment by KIBORG04** [comment]
Заспамь люммоксу этим лицо. Вх со времен диназавров доступно, тут не фиксить, тут надо системы отлова делать. Чтобы такую систему сделать, надо самому качать чит и смотреть как оно работает, а качать чит с ютуба, так ещё и на сс13 (где мамкиных хакеров хватает), это жесть риск.

**Comment by Kiromaka** [comment]
Система отлова whaat ? - делаем слепые зоны - закрытыми окнами tgui без возможности их скрыть - фикс ??????????????????????? ( чел конечно генний, кидать бх с канала на котором читы постит ))))))))

**Comment by AndroBetel** [comment]
какую же ты хуйню тупорылую несешь постоянно господи

**Comment by Kiromaka** [comment]
> какую же ты хуйню тупорылую несешь постоянно господи

ну да - я не отрицаю

**Comment by Oz1Es** [comment]
киборг виртуальную винду запусти, вроде в виндовс это добавили

**Comment by Tankistpidoras228** [comment]
Ахахах, боже. Вы **_НЕ СМОЖЕТЕ НИКАК_** сделать так чтобы чит НЕ работал. До тех пор как игры используют ячейки памяти Вы никак не удалите читы из игры 2002 года в которой все билды относительно одинаковы для взлома.

**Comment by Tankistpidoras228** [comment]
Нет, конечно можно поставить защиту при активации инжектора, и другой параши, но опять таки в этой игре обойти любую защиту можно даже не прибегая к большим программам как "Гидра", или "ИДА". Можно взломать простым "Cheat engine"))

**Comment by Tankistpidoras228** [comment]
Лан, короч. Не интересно с тобой.
